### ISSUE TITLE
An unexpected error has occured

### ISSUE THREAD

**Issue by jthoward09** [avocode/avocode#225]
Every time I try to to open a design within the application I get the "unexpected error has occurred" notification and my design never opens 

https://www.dropbox.com/s/qg6jdvtvapmoiui/Screenshot%202015-04-30%2011.25.02.png?dl=0

-----

```
TypeError: Cannot read property 'title' of undefined
  at Object.module.exports.setLanguage (/Applications/Avocode.app/Contents/Resources/app/node_modules/code-panel/lib/main.js:8:476)
  at /Applications/Avocode.app/Contents/Resources/app/node_modules/code-panel/lib/main.js:5:56
  at Emitter.module.exports.Emitter.emit (/Applications/Avocode.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:71:11)
  at b.module.exports.b.openDocument (/Applications/Avocode.app/Contents/Resources/app/node_modules/avocode/lib/active-document.js:3:261)
  at /Applications/Avocode.app/Contents/Resources/app/node_modules/avocode/lib/user.js:30:84
  at /Applications/Avocode.app/Contents/Resources/app/node_modules/avocode/lib/assets-loader.js:2:390
  at process._tickCallback (node.js:378:11)

```

**Comment by petrbrzek** [comment]
Thanks for reporting the issue. We'll look into it asap.

**Comment by PavelGavlik** [comment]
This should be fixed in 2.2.0. Feel free to reopen this if you run into any problems.
